webpack.config.babel.js

```diff
@@ -52,13 +92,21 @@ export default {
       {from: "install.rdf"},
       {from: "webextension/manifest.json", to: "webextension/"},
       {from: "webextension/**/*.ftl"},
-      {from: "webextension/**/*.html"},
       {from: "webextension/icons/*"},
     ]),
     new webpack.DefinePlugin({
       "process.env": {
         "NODE_ENV": JSON.stringify(NODE_ENV),
       },
     }),
+    new HtmlWebpackPlugin({
+      title: "Lockbox",
```

Comment:
This string is in a bunch of places.  At some point, we really ought to figure out how to parameterize it project-wide.

*eventually*